src/tests/ftest/cart/test_group_np_srv.c

```diff
@@ -35,6 +35,38 @@
 #include "test_group_rpc.h"
 #include "test_group_np_common.h"
 
+/* Callback to process a SWIM message */
+static void
+swim_crt_event_cb(d_rank_t rank, enum crt_event_source src,
+		  enum crt_event_type type, void *arg)
+{
+
+		/* Example output for SWIM CRT_EVT_DEAD on rank #2:
+		 *	 rank = 2, crt_event_source = 1, crt_event_type = 1
+		 *
+		 *		enum crt_event_type {
+		 *			CRT_EVT_ALIVE,
```

Comment:
this function looks like it uses 6 spaces instead of 4/tab.

Reply:
17af1bf3c715de43e313528cc240abd69a8346f2